packages/server/src/subscriptions/websockets.ts

```diff
@@ -210,30 +270,20 @@ export async function handleR4SubscriptionConnection(socket: ws.WebSocket): Prom
         return;
       }
 
-      let tokenPayload: JWTPayload;
-      try {
-        const { payload } = await verifyJwt(token);
```

Comment:
Is there a reason we removed this?

Reply:
It moved to the `verifyWsToken` function since we have to call it for `onBind` or `onUnbind` to clean up the function signatures (we needed to pass both the raw token and the token payload, or invoke it twice if we did it there)